Lushu/src/test/kotlin/lushu/Lushu/LogGenerator/LogGeneratorTest.kt

```diff
@@ -13,7 +14,15 @@ class LogGeneratorTest {
             Paths.get("src", "test", "fixtures", "logs")
                 .toAbsolutePath().toString()
         )
-        lg.run(2)
-        System.setOut(PrintStream(System.out))
+        lg.run(500)
```

Comment:
Please leave the `System.setOut(PrintStream(System.out))`, otherwise all other tests are affected by the interception mechanism.